ToolTip.ios.js

```diff
@@ -1,92 +1,102 @@
 'use strict';
 
-var {
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
   requireNativeComponent,
   TouchableHighlight,
   View,
   NativeModules,
   findNodeHandle,
-} = require('react-native');
-var React = require('react');
-var ToolTipMenu = NativeModules.ToolTipMenu;
-var RCTToolTipText = requireNativeComponent('RCTToolTipText', null);
-
-var propTypes = {
-  actions: React.PropTypes.arrayOf(React.PropTypes.shape({
-    text: React.PropTypes.string.isRequired,
-    onPress: React.PropTypes.func,
-  })),
-  arrowDirection: React.PropTypes.oneOf(['up', 'down', 'left', 'right']),
-  longPress: React.PropTypes.bool,
-  ...TouchableHighlight.propTypes,
-};
-
-var ViewClass = React.createClass({
-  getDefaultProps: function() {
-    return {
-      arrowDirection: 'down'
+} from 'react-native';
+
+const ToolTipMenu = NativeModules.ToolTipMenu;
+const RCTToolTipText = requireNativeComponent('RCTToolTipText', null);
+
+export let isToolTipVisible = false;
```

Comment:
i would put this in the component and expose via ref. this.refs.tooltip.isToolTipVisible.

Reply:
the idea is to use it as an import.

if you have multiple tooltip components and you want to know if there is at least one of those components showing the tooltip without looping though all of them.

does it make sense?

Reply:
> if you have multiple tooltip components and you want to know if there is at least one of those components showing the tooltip without looping though all of them.

That sounds like a state that should live in your application then. It's little weird to keep this global export in a library.idk